### PR TITLE
ci(SW-27133): Cancel previous pipelines

### PR DESCRIPTION
**Pull request by DennisGarding**

### CODE REVIEW

**Review by mitelg — CHANGES_REQUESTED** [review]
branch needs a rebase. the tests are failing because of a test, that is already fixed

**Review by rathmerdominik — APPROVED** [review]

**Review by mitelg — APPROVED** [review]

**Review by mitelg — CHANGES_REQUESTED** [review]
needs a rebase again, because the other PR was merged first :grimacing:

**Review by mitelg — APPROVED** [review]

File: .github/workflows/check-code.yml
Comment (mitelg) [inline .github/workflows/check-code.yml:1]: what is this for an empty file?

**Review by mitelg — CHANGES_REQUESTED** [review]

File: .github/workflows/create-release.yml [hunk 1]
```diff
@@ -7,6 +7,10 @@ on:
         tags:
             - "*"
 
+concurrency:
```
Comment (mitelg) [inline .github/workflows/create-release.yml]: I don't think we need that on create release and create update, as they are only triggered manually or when a tag is pushed

File: .github/workflows/create-update.yml [hunk 2]
```diff
@@ -7,6 +7,9 @@ on:
     push:
         tags:
             - "*"
+concurrency:
```
Comment (mitelg) [inline .github/workflows/create-update.yml]: see above

**Review by mitelg — CHANGES_REQUESTED** [review]

File: .github/workflows/check-code.yml
Comment (mitelg) [reply .github/workflows/check-code.yml:1]: I think you wanted to split the JS and PHP part, right? :thinking:

**Review by mitelg — CHANGES_REQUESTED** [review]

File: .github/workflows/check-code.yml
Comment (DennisGarding) [reply .github/workflows/check-code.yml:1]: it cames from the rebase to main... there it was recreated

**Review by mitelg — APPROVED** [review]

**Review by mitelg — APPROVED** [review]

**Review by ennasus4sun — APPROVED** [review]

**Pull request merged**